clients/admin-ui/cypress/e2e/home.cy.ts

```diff
@@ -0,0 +1,68 @@
+import { stubPlus } from "cypress/support/stubs";
+
+import { RoleRegistry } from "~/types/api";
+
+const ALL_TILES = [
+  "View data map",
+  "Add systems",
+  "View systems",
+  "Configure privacy requests",
+  "Review privacy requests",
+];
+
+const verifyExpectedTiles = (expectedTiles: string[]) => {
+  expectedTiles.forEach((tile) => {
+    cy.getByTestId(`tile-${tile}`);
+  });
+  const remainingTiles = ALL_TILES.filter((t) => !expectedTiles.includes(t));
+  remainingTiles.forEach((item) => {
+    cy.getByTestId(`tile-${item}`).should("not.exist");
+  });
+};
+
+describe("Home page", () => {
+  beforeEach(() => {
+    cy.login();
+  });
+
+  describe("permissions", () => {
```

Comment:
ideally as we build out the UI respecting scopes, every cypress test will have a `permissions` block that goes through and does a `cy.assumeRole` on every relevant role

Reply:
That makes sense. Essentially verify that each role only has access to only what's expected